articleController.js

```diff
@@ -1,5 +1,19 @@
 var articleController = {};
 
 articleController.index = function () {
-  blog.determineData(); 
+  blog.determineData();
+};
+
+articleController.category = function (ctx, next) {
+  var categoryData = function (data) { //rename data to articles?  it is the return of .findByCategory
+    ctx.articles = data;
```

Comment:
Here you are setting the value of `ctx.articles` to `data`, which is actually undefined. I'm guessing you actually want to be getting a value out of ctx and then binding it to something like `articles = ctx.params.category`. It might help to write some documentation for some of these functions stating what you want them to do.

`ctx` has various properties. It might help to set a breakpoint here to inspect them. Either that or consult the [docs](https://visionmedia.github.io/page.js/#context) concerning ctx (context). The docs for page are a bit hard to follow tho... Anyway that's what I did was set a breakpoint and inspect `ctx` and saw that `ctx.params.catagory` is what you are looking for here.